specification/resourcemover/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/resourcemovercollection.json

```diff
@@ -1531,6 +1550,19 @@
           "description": "Gets the value of  next link.",
           "type": "string"
         },
+        "summary": {
```

Comment:
Will the summary be still needed? compared with the previous change: https://github.com/Azure/azure-rest-api-specs/commit/0d41369eaba5da010d672625288ca9d6d5b2fdb2

Reply:
@innosam You may need to take a look at it. It will cause lots of error in CI.

Reply:
yes, I am looking into the issue, looks like the earlier there was a breaking issue, which has been fixe by durgesh.

Reply:
I have fixed the issue now, and pr is green.